pkg/flare/archive.go

```diff
@@ -225,11 +225,17 @@ func createArchive(zipFilePath string, local bool, confSearchPaths SearchPaths,
 	if err != nil {
 		log.Errorf("Could not write typeperf data: %s", err)
 	}
+
 	err = zipCounterStrings(tempDir, hostname)
 	if err != nil {
 		log.Errorf("Could not write counter strings: %s", err)
 	}
 
+	err = zipEventLogs(tempDir, hostname)
```

Comment:
Could we rename it to `zipWindowsEventLogs` ?

Reply:
It follows the convention of zipCounterStrings. Won't change.

Reply:
I understand but `Logs` is a whole datadog product which makes it confusing I thinks